--- old
+++ new
@@ -12,0 +12,18 @@
+
+Personalised qr codes for property 
+An MX card can contain a routine to contact the card owner before responding to the card discoverer
+
+Cards can be time-limited and scheduled
+
+Cards can be encrypted 
+
+Plain cards are free
+
+Schedule time limited and encryptedcards fist more 
+
+App has an on board card with the owner's personal identity layer. 
+
+Cards can be shared with agents and Chatbots 
+
+
+