--- old
+++ new
@@ -24,1 +24,1 @@
- I am not pointing out flaws in AI or websites, but pointing out design flaws in MX ( new technology role) 
+ I am not pointing out flaws in AI AI is brilliant. I use it a lot or websites, but pointing out design flaws in MX ( new technology role) 
@@ -26,0 +26,1 @@
+ Websites designed for human attention can be  difficult to navigate for humans as well as machines, design for both. 
@@ -27,0 +28,5 @@
+ A great refresher is don't make me think by Steve krug. Written in 2002. If we had listen to him this book would have been much shorter. 
+ 
+ 
+ 
+ 